helpers/cfgsetup.py

```diff
@@ -8,23 +8,23 @@
 with open(hostConfigFileName, "r") as file:
     hostConfigFile = file.readlines()
 currentSSID = hostConfigFile[2][5:].strip()
-currentPassphrase = hostConfigFile[10][15:].strip()
+#currentPassphrase = hostConfigFile[10][15:].strip()
 print("hostapd configuration - SSID: " + currentSSID)
-print("hostapd configuration - Passphrase: " + currentPassphrase)
+#print("hostapd configuration - Passphrase: " + currentPassphrase)
 
 # get SSID and passphrase from user configuration file
-nwConfigFileName = "/boot/_naturewatch-configuration.txt"
-with open(nwConfigFileName, "r") as file:
-    nwConfigFile = file.readlines()
-configFileSSID = nwConfigFile[1].strip()
-configFilePassphrase = nwConfigFile[3].strip()
-if "myNatureWatchCam" in configFileSSID :
-      unique_id = subprocess.check_output("sed -n 's/^Serial\s*: 0*//p' /proc/cpuinfo", shell=True)
-      configFileSSID = "myNatureWatchCam-" + unique_id.strip().decode('utf-8')
-      print("Wifi Updated to unique name")
-
-print("Boot configuration - SSID: " + configFileSSID)
-print("Boot configuration - Passphrase: " + configFilePassphrase)
+#nwConfigFileName = "/boot/_naturewatch-configuration.txt"
+#with open(nwConfigFileName, "r") as file:
+#    nwConfigFile = file.readlines()
+#configFileSSID = nwConfigFile[1].strip()
+#configFilePassphrase = nwConfigFile[3].strip()
+#if "myNatureWatchCam" in configFileSSID :
+unique_id = subprocess.check_output("sed -n 's/^Serial\s*: 0*//p' /proc/cpuinfo", shell=True)
+configFileSSID = "myNatureWatchCam-" + unique_id.strip().decode('utf-8')
```

Comment:
I think this needs to be "MyNaturewatch-", at the moment that's what the AUTOHOTSPOT name is.

Reply:
good catch, will update